src/bindings/python/tests/test_frontend/test_frontend_onnx.py

```diff
@@ -395,15 +395,15 @@ def check_attribute(context, name, expected_value, dtype):
 
         check_attribute(node, "attribute_i32", 10, float)
         check_attribute(node, "attribute_i64", 10, float)
-        check_attribute(node, "attribute_str", "string", np.str)
+        check_attribute(node, "attribute_str", "string", str)
         check_attribute(node, "attribute_f32", 10, int)
         check_attribute(node, "attribute_f64", 10, int)
         check_attribute(node, "attribute_bool", True, bool)
         check_attribute(node, "attribute_type", Type.i32, Type)
 
         check_attribute(node, "attribute_list_i32", [1., 2., 3.], float)
         check_attribute(node, "attribute_list_i64", [1., 2., 3.], float)
-        check_attribute(node, "attribute_list_str", ["a", "b", "c"], np.str)
+        check_attribute(node, "attribute_list_str", ["a", "b", "c"], str)
         check_attribute(node, "attribute_list_f32", [1, 2, 3], int)
         check_attribute(node, "attribute_list_f64", [1, 2, 3], int)
```

Comment:
It looks like some checks have typo's in their names - "f" and "i" probably should be replaced.
check_attribute(node, "attribute_i32", 10, float)
check_attribute(node, "attribute_f32", 10, int)

Reply:
Good point! This can be done in separate PR.

Reply:
The strings are attributes taken from a custom `onnx` model above. They should not be changed, the fourth parameter to `check_attribute` (`dtype`) specifies what type the result should be cast to. They do not specify the same type by design.